src/borg/algorithms/crc32_dispatch.c

```diff
@@ -75,6 +82,7 @@
 static uint32_t
 crc32_clmul(const uint8_t *src, long len, uint32_t initial_crc)
 {
+    (void)src; (void)len; (void) initial_crc;
```

Comment:
not sure what that even means, but maybe distribute blanks more consistently. :)